### DISCUSSION TITLE
What is the unit of transformation width and height in ImageRun

### DISCUSSION THREAD

**Discussion by keogami** [dolanmiu/docx#2709] [Q&A]
What is the unit of measurement used for width and height of the ImageRun. I thought it was Twip but that seems to be wrong.

It seems that `inch * 96` gives the correct result in the rendered docx.

Also, how does `scale` relate to transformation?

**Comment by anti-the-social** [comment]
Hello!
Transformation.width and transformation.height seems to be in pixels. Which is 0.75*points
Where did you find scale for the ImageRun?
